src/bgw/job_stat.c

```diff
@@ -565,7 +602,23 @@ ts_bgw_job_stat_next_start(BgwJobStat *jobstat, BgwJob *job, int32 consecutive_f
 		return DT_NOBEGIN;
 
 	if (jobstat->fd.consecutive_crashes > 0)
+	{
+		if (!ts_flags_are_set_32(jobstat->fd.flags, LAST_CRASH_REPORTED))
+		{
+			/* Update the errors table regarding the crash */
+			FormData_job_error jerr = { 0 };
+			jerr.error_data = NULL;
+			jerr.start_time = jobstat->fd.last_start;
+			jerr.finish_time = ts_timer_get_current_timestamp();
+			jerr.pid = -1;
+			jerr.job_id = jobstat->fd.id;
```

Comment:
Nit: We support C99, and if you provide an initializer, the remaining fields will automatically be initialized to zero.

```suggestion
			const FormData_job_error jerr = {
			.error_data = NULL,
			.start_time = jobstat->fd.last_start,
			.finish_time = ts_timer_get_current_timestamp(),
			.pid = -1,
			.job_id = jobstat->fd.id,
		};
```